workspace/lib/lita/handlers/teamcity.rb

```diff
@@ -130,6 +130,15 @@ class Teamcity < Handler
           t('help.queue.syntax_wild') => t('help.queue.desc_wild')
         }
       )
+      
+      route(
+        /.*artifacts(.*)/,
```

Comment:
why is this regex different than the other routes?

Reply:
I just want one route for this function, when user don't pass build_number, we will get latest build (success and finished) to get artifacts.For the consistency routes, I will split this to two route like queue/running command